sdk/include/psdk/shobjidl.idl

```diff
@@ -4768,4 +4768,40 @@ interface IItemNameLimits : IUnknown
         [out] int *piMaxNameLen);
 };
 
+/*****************************************************************************
+ * IUserAssist interface
+ */
+[
+    uuid(dd313e04-feff-11d1-8ecd-0000f87a470c),
```

Comment:
This is CLSID_UserAssist, it should be IID_IUserAssist, which is `DD313E05-FEFF-11D1-8ECD-0000F87A470C` according to https://www.geoffchappell.com/studies/windows/ie/browseui/interfaces/iuserassist.htm

Reply:
Indeed. Thanks @learn-more. :+1: 
Now that error does no longer appear for me in the log.
But it still doesn't work properly.
Judging by the log, now it needs `ShellTaskSheduler`, same as MS shimgvw.dll from CORE-17337.
I think I can send that in another PR. Since that one is documented by MS: https://docs.microsoft.com/en-us/windows/win32/api/shobjidl_core/nn-shobjidl_core-ishelltaskscheduler, unlike `UserAssist`, it will be more easy to stubplement that.